### DISCUSSION TITLE
Show contributions on mobile

### DISCUSSION THREAD

**Discussion by tobyab** [community/community#6149] [Mobile] [answered]
There is an iOS widget that does this (not sure about android) but it would be really helpful if you were able to see contributions on the github app.

**Comment by FelixHerrmann** [comment]
Duplicate of #5719

**Comment by taki-on** [comment, marked as answer]
I think this is a great idea. I will pass the idea to the team and track internally. Thank you for the suggestion @ItsTobez and @FelixHerrmann !
